lib/Word_Lib/Ancient_Numeral.thy

```diff
@@ -0,0 +1,235 @@
+(*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
```

Comment:
Do we need to worry about the license header here?

Reply:
It's mostly just material that has been moved around and/or slightly adjusted, so that header is accurate. But this is just imported from the AFP anyway, almost none of the changes in Word_Lib are ours.